__tests__/app.test.js

```diff
@@ -135,3 +135,50 @@ describe('/api/articles/:article_id/comments', () => {
 });
 
 
+describe('/api/articles/:article_id/comments', () => {
+    test('POST request reponds with 201 and the added comment', () => {
+        const testComment =   {
+            username: 'thebear22',
```

Comment:
I kept getting the error, 'Insert or update on table violates foreign key constraint', so I added a user with the username and just a placeholder 'name' for the name column. I couldn't find a way for it to work without adding a user.

Reply:
Ideally, only registered users should be able to post a comment. I understand you have added thebear22 but what if another user that doesn't exist tries to post a comment? This will indeed violate the foreign key restraint and we should handle this error